### PR TITLE
Provides conan recipe file to build artifacts

### PR DESCRIPTION
**Pull request by deekshith-elear**
Adds conan recipe file to this repo

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    def build(self):
+        cmake = CMake(self)
+        if (self.settings.os == "Android"):
+            cmake.definitions["Platform"] = "android"
+        cmake.configure(source_folder=".")
+        cmake.build()
+
+    def package(self):
+        self.copy("*.h", dst="include", src="include")
+        self.copy("*", dst="lib", keep_path=False)
```
Comment (manavkumarm) [inline conanfile.py]: Does this also copy header files?

**Review by manavkumarm — CHANGES_REQUESTED** [review]

File: conanfile.py (hunk 1 above)
Comment (deekshith-elear) [reply conanfile.py]: > Does this also copy header files?

Yes, it copies `header files from source directory to package ` and ` libraries from build directory to lib directory in package`

File: conanfile.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class JanssonlibConan(ConanFile):
+    name = "jansson"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of janssonlib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
```
Comment (manavkumarm) [inline conanfile.py]: Fix spacing everywhere as per httpserver feedback

**Review by manavkumarm — CHANGES_REQUESTED** [review]

File: conanfile.py (hunk 2 above)
Comment (deekshith-elear) [reply conanfile.py]: handled

File: conanfile.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+class JanssonlibConan(ConanFile):
+    name = "jansson"
+    version = "0.1"
+    license = "<Put the package license here>"
+    author = "<Put your name here> <And your email here>"
+    url = "<Package recipe repository url here, for issues about the package>"
+    description = "<Description of janssonlib here>"
+    topics = ("<Put some tag here>", "<here>", "<and here>")
+    settings = "os", "compiler", "build_type", "arch"
+    options = { "shared": [True, False] }
```
Comment (manavkumarm) [inline conanfile.py:12]: Pending style fix: `[ True, False ]` - space after [, space before ]

**Review by manavkumarm — APPROVED** [review]

File: conanfile.py (hunk 3 above)
Comment (deekshith-elear) [reply conanfile.py:12]: handled

**Pull request merged**